Sources/Sentry/SentryClient.m

```diff
@@ -74,18 +69,15 @@ @implementation SentryClient
 - (_Nullable instancetype)initWithOptions:(SentryOptions *)options
                          didFailWithError:(NSError *_Nullable *_Nullable)error {
```

Comment:
Can we just remove the `didFailWithError` part? This is no longer used here. Also removes the check later in the static API

Reply:
done